src/WorkOS.net/Services/SSO/Enums/Type.cs

```diff
@@ -8,7 +8,7 @@
     /// Constants that enumerate available <see cref="Connection"/> types.
     /// </summary>
     [JsonConverter(typeof(StringEnumConverter))]
-    public enum ConnectionType
+    public enum Type
```

Comment:
We can keep the enum name as `ConnectionType`, it's just the field name that needs to change.

Reply:
Do we want to change Profile.ConnectionType to Profile.Type as well then?

Reply:
We do, good call!